x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/open_timeline.cy.ts

```diff
@@ -23,48 +24,41 @@ import { visit } from '../../../tasks/navigation';
 import {
   markAsFavorite,
   openTimelineById,
+  openTimelineByIdFromOpenTimelineModal,
   openTimelineFromSettings,
   pinFirstEvent,
   refreshTimelinesUntilTimeLinePresent,
 } from '../../../tasks/timeline';
-
 import { TIMELINES_URL } from '../../../urls/navigation';
 
+const defaultTimeline = getTimeline();
+
 describe('Open timeline', { tags: ['@serverless', '@ess'] }, () => {
-  describe('Open timeline modal', () => {
-    before(function () {
-      login();
-      visit(TIMELINES_URL);
-      createTimeline(getTimeline())
-        .then((response) => response.body.data.persistTimeline.timeline.savedObjectId)
-        .then((timelineId: string) => {
-          refreshTimelinesUntilTimeLinePresent(timelineId)
-            // This cy.wait is here because we cannot do a pipe on a timeline as that will introduce multiple URL
-            // request responses and indeterminism since on clicks to activates URL's.
-            .then(() => cy.wrap(timelineId).as('timelineId'))
-            // eslint-disable-next-line cypress/no-unnecessary-waiting
-            .then(() => cy.wait(1000))
-            .then(() =>
-              addNoteToTimeline(getTimeline().notes, timelineId).should((response) =>
-                expect(response.status).to.equal(200)
-              )
-            )
-            .then(() => openTimelineById(timelineId))
-            .then(() => pinFirstEvent())
-            .then(() => markAsFavorite());
-        });
-    });
+  beforeEach(function () {
+    login();
+    visit(TIMELINES_URL);
+    deleteTimelines();
+    createTimeline(defaultTimeline)
+      .then((response) => response.body.data.persistTimeline.timeline.savedObjectId)
+      .then((timelineId: string) => {
+        cy.wrap(timelineId).as('timelineId');
+        refreshTimelinesUntilTimeLinePresent(timelineId);
```

Comment:
just putting a comment here, the `refreshTimelinesUntilTimeLinePresent` has a long documentation that is not accurate anymore, so could be removed entirely.
Honesty the name of the function is even misleading, there isn't any retry anymore, so maybe we should change it, or get rid of it entirely?

But maybe you'd prefer doing that in another PR...

Reply:
I may do it in a separate PR. I think there should be an infrastructure cleanup PR for a lot of these since they go across files